examples/sklearnex/incremental_pca_dpctl.py

```diff
@@ -13,14 +13,26 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 # ==============================================================================
+
+# sklearnex IncrementalPCA example for GPU offloading with dpctl.tensor:
+#    SKLEARNEX_PREVIEW=YES python ./incremental_pca_dpctl.py
+
 import dpctl
 import dpctl.tensor as dpt
 
+# Just directly import estimator from sklearnex
```

Comment:
I'd try to make the commentary about the importing to the users in a single block of comment, ideally without ```##``` (not sure why that is there. Also I would write it in complete sentences due to the audience.

Reply:
Maybe this is not the best solution.  I wanted to separate comments from the code that was commented
Probably it is worth rethinking how to do it better without double ## sign